Podfile.lock

```diff
@@ -21,4 +21,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 9db81edec5b4deeb01978486713076bbb76db6fa
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.8.0
```

Comment:
Can you drop this change from your PR? We shouldn't be changing to CocoaPods version here.